runtime/dancebox/src/lib.rs

```diff
@@ -767,6 +772,105 @@ impl pallet_services_payment::Config for Runtime {
     type WeightInfo = pallet_services_payment::weights::SubstrateWeight<Runtime>;
 }
 
+pub struct DanceboxContainerChainManagerOrRootOrigin<T, RootOrigin> {
```

Comment:
It might be easier to implement something like this in the pallet, and wrap it in a struct: https://github.com/paritytech/polkadot-sdk/blob/1f2ccaea05c8508f5cc916f8ae0ba60c102c5cf3/polkadot/runtime/common/src/paras_registrar/mod.rs#L540

Reply:
@tmpolaczyk check 160e00c2f201680ac579c73b0cbe66982c077212, revert it if you find it less clean, I do find it cleaner than what we had